dask/tests/test_base.py

```diff
@@ -666,15 +767,94 @@ def test_persist_array():
     assert len(y.dask) == y.npartitions
 
 
+@pytest.mark.skipif("not da")
+def test_persist_array_change_name():
+    a = da.zeros(4, chunks=2)
+    rebuild, args = a.__dask_postpersist__()
+    dsk = {("b", 0): np.array([1, 2]), ("b", 1): np.array([3, 4])}
+    b = rebuild(dsk, *args, name="b")
+    assert isinstance(b, da.Array)
+    assert b.name == "b"
+    assert b.__dask_keys__() == [("b", 0), ("b", 1)]
+    np.testing.assert_array_equal(b, [1, 2, 3, 4])
```

Comment:
Similar to using `dd.utils.assert_eq`, we tend to use `da.utils.assert_eq` when comparing Dask arrays to other Dask arrays or array-like objects

Reply:
fixed